theurgy.lic

```diff
@@ -729,12 +725,24 @@ class TheurgyActions
     else
       DRCI.get_item_safe('incense', @theurgy_supply_container)
     end
+    
+    unless exists?('flint')
```

Comment:
This will attempt to tap 'flint'. When would this return `false` and then line 731 would succeed?

Reply:
If flint is in your eddy/portal its not on your person so you can't just tap thing, you have to tap thing in your eddy.  (Think portable wormhole to a vault.  You can reach through the eddy to a vault somewhere else but only if you specify). If you can just tap (because its in a normal storage) you don't have to pick it up.  If you can't because its in your eddy, you need to grab it with get_item_safe which specifies portal.

Reply:
Could you add a comment that this is to work around eddies?

Reply:
Adding!